### DISCUSSION TITLE
思源笔记应用更新问题

### DISCUSSION THREAD

**Discussion by Mask-Vip** [1Panel-dev/1Panel#5058] [Q&A]
### 1Panel 版本

社区版：1.10.6-list

### 问题描述

在更新思源笔记3.0.13版本后，提示失败，失败原因是容器不存在。
然后重建容器后，思源笔记版本会回退到3.0.11，虽然在应用商店里显示3.0.13版本了，但是容器界面显示的是3.0.11，即使下载了3.0.13版本的镜像，只要一重建还是会强制下载3.0.11的镜像进行重建。
后来发现安装目录下的 docker-compose.yml 文件里就是 3.0.11 最初安装的版本号，应该是升级后没有刷新这个文件。
手动修改docker-compose.yml文件内的镜像版本号后在重建的时候就可以使用3.0.13的镜像重建了。

### 重现步骤

安装3.0.11的思源笔记应用，升级应用后重建应用。

**Comment by wanghe-fit2cloud** [comment]
感谢反馈。
出现上述问题的原因可能是升级时由于镜像拉取超时导致的。
